src/D-FSL-1.0.xml

```diff
@@ -0,0 +1,380 @@
+             Lizenz nachkommen und die Rechte der Dritten aus dieser Lizenz nicht beeinträchtigt werden.
+             Insbesondere dürfen Sie für die Überlassung des Programms oder sonstige Leistungen ein Entgelt
+             verlangen.</p>
+        </li>
+        <li>
+          <b>(3)</b>
+          <p>Diese Lizenz verpflichtet Sie nicht, das Programm an Dritte weiterzugeben. Es steht Ihnen frei zu
+             entscheiden, wem Sie das Programm zugänglich machen. Sie dürfen aber die weitere Nutzung durch
+             Dritte nicht durch den Einsatz technischer Schutzmaßnahmen, insbesondere durch den Einsatz von
+             Kopierschutzvorrichtungen jeglicher Art, verhindern oder erschweren. Eine passwortgeschützte
+             Zugangsbeschränkung oder die Nutzung in einem Intranet wird nicht als technische
+             Schutzmaßnahme angesehen.</p>
+          <p>§ 9 Text der Lizenz</p>
+        </li>
+        <li>
+          <b>(1)</b>
+          <p>Diese Lizenz ist in deutscher und englischer Sprache abgefasst. Beide Fassungen sind gleich
+             verbindlich. Es wird unterstellt, dass die in der Lizenz verwandten Begriffe in beiden
+             Fassungen dieselbe Bedeutung haben. Ergeben sich dennoch Unterschiede, so ist die Bedeutung
+             maßgeblich, welche die Fassungen unter Berücksichtigung des Ziels und Zwecks der Lizenz am
+             besten miteinander in Einklang bringt.</p>
+        </li>
+        <li>
+          <b>(2)</b>
+          <p>Der Lizenzrat der Deutschen Freien Software Lizenz kann mit verbindlicher Wirkung neue Versionen
+             der Lizenz in Kraft setzen, soweit dies erforderlich und zumutbar ist. Neue Versionen der
+             Lizenz werden auf der Internetseite http://www.d-fsl.de mit einer eindeutigen Versionsnummer
+             veröffentlicht. Die neue Version der Lizenz erlangt für Sie verbindliche Wirkung, wenn Sie von
+             deren Veröffentlichung Kenntnis genommen haben. Gesetzliche Rechtsbehelfe gegen die Änderung
+             der Lizenz werden durch die vorstehenden Bestimmungen nicht beschränkt.</p>
+        </li>
+        <li>
+          <b>(3)</b>
+          <p>Sie dürfen diese Lizenz in unveränderter Form vervielfältigen, verbreiten und öffentlich
+             zugänglich machen.</p>
+        </li>
+      </list>
+      <p>§ 10 Anwendbares Recht</p>
+      <p>Auf diese Lizenz findet deutsches Recht Anwendung.</p>
+      <p>Anhang: Wie unterstellen Sie ein Programm der Deutschen Freien Software Lizenz? 
+        <br/>Um jedermann den Abschluss dieser Lizenz zu ermöglichen, wird empfohlen, das Programm mit folgendem
+             Hinweis auf die Lizenz zu versehen: 
+      </p>
+      <p>"Copyright (C) 20[jj] [Name des Rechtsinhabers].</p>
+      <p>Dieses Programm kann durch jedermann gemäß den Bestimmungen der Deutschen Freien Software Lizenz genutzt
+         werden.</p>
+      <p>Die Lizenz kann unter http://www.d-fsl.de abgerufen werden."</p>
```

Comment:
Upstream is inconsistent here, using [no wrapping chars in the HTML][1] and [`<>` in the text][2].   I recommend using `<alt>` or `<optional>` to cover both cases, and have no opinion on which form should be canonical.

[1]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/deutsche-freie-software-lizenz
[2]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/D-FSL-1_0_de.txt/view